include/functions.h

```diff
@@ -2171,21 +2150,19 @@ void func_800F5B58(void);
 void func_800F5BF0(u8 arg0);
 void func_800F5C64(u16);
 void func_800F5C2C(void);
-// ? func_800F5CF8(?);
-// ? func_800F5E18(?);
+void func_800F5E18(u8 seqIdx, u16 seqId, u8 fadeTimer, s8 arg3, s8 arg4);
 void func_800F5E90(u8);
-void func_800F6114(f32);
-void func_800F6268(f32, u16);
+void func_800F6114(f32 dist);
+void func_800F6268(f32 dist, u16);
 void func_800F64E0(u8 arg0);
 void func_800F6584(u8 arg0);
-void func_800F66C0(s8 echo);
-void func_800F66DC(s8);
+void Audio_SetEnvReverb(s8 reverb);
+void Audio_SetCodeReverb(s8 reverb);
 void func_800F6700(s8 outputMode);
 void func_800F67A0(u8);
 void func_800F6828(u8);
-void func_800F68BC(s8);
-// ? func_800F68D4(?);
-// ? func_800F691C(?);
+void Audio_SetCutsceneFlag(s8 flag);
+void Audio_PlaySoundIfNotDemo(u16 sfxId);
```

Comment:
```suggestion
void Audio_PlayCutsceneSound(u16 sfxId);
```
at the very least Demo -> Cutscene but I think this name looks better too
`Audio_PlayCutsceneOnlySound` may be more explicit if the cutscene flag really means "currently in a cutscene", idk

Reply:
Oops, forgot to change that. Audio_PlayCutsceneSound would indicate the opposite thing, but I'll change it to PlaySoundIfNotInCutscene. It's only called from one place so the name doesn't have to be perfect.